src/api_server/src/parsed_request.rs

```diff
@@ -1014,11 +941,10 @@ pub(crate) mod tests {
     fn test_try_from_patch_mmds() {
         let (mut sender, receiver) = UnixStream::pair().unwrap();
         let mut connection = HttpConnection::new(receiver);
+        let body = "{}";
```

Comment:
NIT:

Since the body is basically "{}", I believe we could send it directly as a parameter.

http_request("PATCH", "/mmds", **"{}"**).as_bytes()